pkg/utils/rfctime/rfc3339_test.go

```diff
@@ -99,3 +99,195 @@ func TestRFC3339(t *testing.T) {
 		})
 	})
 }
+func Test_StringWithLocalTimeZone(t *testing.T) {
+	t.Run("it should return local time zone when passed format without local time zone", func(t *testing.T) {
+		s := "2024-04-02"
+		testee := rfctime.RFC3339(try.To(time.Parse(time.DateOnly, s)).OrFatal(t))
+
+		actual, err := testee.StringWithLocalTimeZone()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		expected := "2024-04-02T09:00:00+09:00"
+		if actual != expected {
+			t.Errorf("unmatch: (actual, expected) = (%s, %s)", actual, expected)
+		}
+	})
+}
+
+func Test_ParseLooseRFC3339(t *testing.T) {
+	type when struct {
+		args []string
+	}
+	type then struct {
+		expected []time.Time
+	}
+
+	theory := func(when when, then then) func(*testing.T) {
+		return func(t *testing.T) {
+			for i, w := range when.args {
+				testee, err := rfctime.ParseLooseRFC3339(w)
+				if err != nil {
+					t.Fatal(err)
+				}
+				expectdRFC3339 := rfctime.RFC3339(then.expected[i])
+
+				if !testee.Time().Equal(then.expected[i]) {
+					t.Errorf("unmatch: as time: (actual, expected) = (%+v, %+v)", testee, then.expected[i])
+				}
+
+				if !testee.Equiv(expectdRFC3339) {
+					t.Errorf("unmatch: as RFC3339: (actual, expected) = (%+v, %+v)", testee, expectdRFC3339)
+				}
+			}
+		}
+	}
+
+	t.Run("it should parse when passed RFC3339DateNano format", theory(
+		when{
+			args: []string{
+				"2024-04-22T12:34:56.987654321+07:00",
+				"2024-04-22 12:34:56.987654321+07:00",
+				"2024-04-22T12:34:56.987654321",
+				"2024-04-22 12:34:56.987654321",
+			},
+		},
+		then{
+			expected: []time.Time{
+				time.Date(
+					2024, 4, 22, 12, 34, 56, 987654321,
+					time.FixedZone("+07:00", int((7*time.Hour).Seconds())),
+				),
+				time.Date(
+					2024, 4, 22, 12, 34, 56, 987654321,
+					time.FixedZone("+07:00", int((7*time.Hour).Seconds())),
+				),
+				time.Date(
+					2024, 4, 22, 12, 34, 56, 987654321,
+					time.FixedZone("+09:00", int((9*time.Hour).Seconds())),
```

Comment:
This test will fail when someone runs the test in London, for example. `+09:00` is for only Far-East Asia, not worldwide.

The fact to be tested is "if timezone is omitted, it is failback to local timezone". Testcase also should be so.